tests/test_extensions/test_ext_autosectionlabel.py

```diff
@@ -98,3 +98,78 @@ def test_autosectionlabel_maxdepth(app):
     assert re.search(html, content, re.DOTALL)
 
     assert "WARNING: undefined label: 'linux'" in app.warning.getvalue()
+
+
+@pytest.mark.sphinx('html', testroot='ext-autosectionlabel-full-reference')
+def test_autosectionlabel_full_reference(app):
```

Comment:
Can you also add tests for the LaTeX output and texinfo/man if the extension is properly supported? (I think LaTeX should be fine but I'm not sure for texinfo/man).